bindings/wasm/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.1 - 2023-07-25
+
+### Fixed
+
+- `destroyClient` is not a function;
```

Comment:
This feels a bit weird in a changelog tbh, can you think of a more descriptive sentence for what was actually fixed?